/test/unit/specs/Hello.spec.js has missing semicolons - reports 7 eslint errors with AirBNB preset

If you do a fresh project with `vue`, version 2.2.2 and answer like this:

```
vue init webpack

? Generate project in current directory? Yes
? Project name vuejs-test-1
? Project description A Vue.js project
? Author Duncan Lock <duncan.lock@gmail.com>
? Use ESLint to lint your code? Yes
? Pick an ESLint preset AirBNB
? Setup unit tests with Karma + Mocha? Yes
? Setup e2e tests with Nightwatch? No

   vue-cli · Generated "vuejs-test-1".

```

then run `npm instal`, then run `npm run lint`, you get this output:

```
 $ npm run lint

> vuejs-test-1@1.0.0 lint /home/duncan/dev/vuejs-test-1
> eslint --ext .js,.vue src test/unit/specs

/home/duncan/dev/vuejs-test-1/test/unit/specs/Hello.spec.js
   1:22  error  Missing semicolon       semi
   2:41  error  Missing semicolon       semi
   8:28  error  Missing trailing comma  comma-dangle
   9:16  error  Missing semicolon       semi
  10:85  error  Missing semicolon       semi
  11:5   error  Missing semicolon       semi
  12:3   error  Missing semicolon       semi

✖ 7 problems (7 errors, 0 warnings)
```

Which looks accurate to me? This is what my `Hello.spec.js` looks like:

```js
import Vue from 'vue'
import Hello from 'src/components/Hello'

describe('Hello.vue', () => {
  it('should render correct contents', () => {
    const vm = new Vue({
      template: '<div><hello></hello></div>',
      components: { Hello }
    }).$mount()
    expect(vm.$el.querySelector('.hello h1').textContent).to.contain('Hello World!')
  })
})
```

Presumably this isn't compatible with the eslint AirBNB preset?

--- a/template/test/unit/specs/Hello.spec.js
+++ b/template/test/unit/specs/Hello.spec.js
@@ -1,12 +1,12 @@
-import Vue from 'vue'
-import Hello from 'src/components/Hello'
+import Vue from 'vue'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+import Hello from 'src/components/Hello'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
 
 describe('Hello.vue', () => {
   it('should render correct contents', () => {
     const vm = new Vue({
       el: document.createElement('div'),
-      render: (h) => h(Hello)
-    })
-    expect(vm.$el.querySelector('.hello h1').textContent).to.equal('Hello Vue!')
-  })
-})
+      render: (h) => h(Hello){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+    }){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+    expect(vm.$el.querySelector('.hello h1').textContent).to.equal('Hello Vue!'){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+  }){{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+}){{#if_eq lintConfig "airbnb"}};{{/if_eq}}

